surveyor.py

```diff
@@ -113,27 +125,30 @@ def cli(ctx, prefix, hostname, profile, days, minutes, product, username, iocfil
                 for ioc in data:
                     ioc = ioc.strip()
                     query = f"{ioctype}:{ioc}"
-                    results = utils.process_search(cb_conn, query, base_query)
+                    results = utils.process_search(conn, query, base_query)
                     click.echo(f"-->{ioc}")
                     utils.write_csv(writer, results, ioc, 'ioc')
 
     # run search against definition files and write to csv
     if deffile is not None or defdir is not None:
         results_set = set()
         for definitions in definition_files:
-            click.echo(f"Processing definition file for {definitions}")
+            click.echo(f"\033[96m Processing definition file for {definitions} \033[0m")
             basename = os.path.basename(definitions)
             source = os.path.splitext(basename)[0]
 
             with open(definitions, 'r') as file:
                 programs = json.load(file)
                 for program, criteria in programs.items():
-                    nested_results = utils.nested_process_search(criteria, cb_conn, base_query)
-                    click.echo(f"-->{program}: {len(nested_results)} results")
+                    nested_results = utils.nested_process_search(criteria, conn, base_query)
+                    if len(nested_results) > 0:
+                        click.echo(f"\033[92m -->{program}: {len(nested_results)} results \033[0m")    
+                    else:                     
```

Comment:
🗣️ Maybe this makes more sense when it runs, but is the output in this else block helpful and accurate?  Would it be printing out 0 results every time?

Reply:
It would be printing out 0 results every time but there would be no color associated with it. Currently surveyor prints out all the programs it's looking for and the number of results returned. By splitting it into this if/else statement I was able to color the output that had results and draw the users eyes to those.